setup_ngrok.py

```
@@ -0,0 +1,143 @@
+import subprocess
+import time
+from datetime import datetime
+
+def start_ngrok():
+    """Start ngrok tunnel for port 8000"""
+    print("🌐 Starting ngrok tunnel for port 8000...")
+
+    try:
+        # Start ngrok in the background
+        process = subprocess.Popen(
+            ["ngrok", "http", "8000"],
```

Ruff (0.13.1)

Starting a process with a partial executable path

(S607)